### PR TITLE
feat: adding the `starlark run` repository to maintain this data after an APIC restart

### PR DESCRIPTION
**Pull request by leoporoli**
## Description
adding the `starlark run` repository to have this data after an APIC restart

## Is this change user facing?
NO

### CODE REVIEW

**Comment by laurentluce** [comment]
I will review this change tomorrow.  I need a bit more time to ramp-up on the concept of APIC repositories.

File: core/server/api_container/server/api_container_service.go [hunk 1]
```diff
@@ -963,6 +1001,18 @@ func (apicService *ApiContainerService) runStarlark(
...
+
+				if err := apicService.saveStarlarkRun(
+					packageId,
+					serializedParams,
+					serializedStarlark,
+					parallelism,
+					mainFunctionName,
+					experimentalFeatures,
+				); err != nil {
+					logrus.Errorf("The Starlark code wass successfully executed but something failed when trying to save the 'run' info in the enclave's database. Error was: \n%v", err.Error())
```
Comment (laurentluce) [inline core/server/api_container/server/api_container_service.go]:
```suggestion
					logrus.Errorf("The Starlark code was successfully executed but something failed when trying to save the 'run' info in the enclave's database. Error was: \n%v", err.Error())
```

File: api/typescript/src/engine/rest_api_bindings/types.d.ts [hunk 2]
```diff
@@ -3,1076 +3,1610 @@
  * Do not make direct changes to the file.
  */
 
-
 export interface paths {
-  "/engine/info": {
```
Comment (leoporoli) [inline api/typescript/src/engine/rest_api_bindings/types.d.ts:8]: these changes were auto-generated when I ran the `scripts/build.sh` script

**Review by laurentluce — APPROVED** [review]
@tedim52 Should you take a look since you made some recent changed related to the Starlark run?

**Comment by tedim52** [comment]
Hey @leoporoli ! curious, why is the starlark run information required after APIC restart? Do you plan on using it to run the same starlark against an enclave to restart it?

**Comment by leoporoli** [comment]
> Hey @leoporoli ! curious, why is the starlark run information required after APIC restart? Do you plan on using it to run the same starlark against an enclave to restart it?

Hey, @tedim52, well I realized that we had to persist it because the `enclave list` page in the cloud UI is making calls to the `GetStarlarkRun` endpoint to get the `package ID` value, so it returns an empty value after an APIC restart.

**Pull request merged**